src/runtime/components/elements/ButtonGroup.ts

```diff
@@ -12,11 +13,12 @@ const buttonConfig = mergeConfig<typeof button>(appConfig.ui.strategy, appConfig
 const buttonGroupConfig = mergeConfig<typeof buttonGroup>(appConfig.ui.strategy, appConfig.ui.buttonGroup, buttonGroup)
 
 export default defineComponent({
+  name: 'ButtonGroup',
   inheritAttrs: false,
   props: {
     size: {
       type: String as PropType<ButtonSize>,
-      default: null,
+      default: buttonConfig.default.size,
```

Comment:
The `default` shouldn't be set here, otherwise it will override the default size of children. For example, the default size of the `Input` might not be the same as the one of the `Button`.

Reply:
Resolved with latest commit